crypto/s390xcap.c

```diff
@@ -82,7 +82,9 @@ void OPENSSL_cpuid_setup(void)
     /* set a bit that will not be tested later */
     OPENSSL_s390xcap_P.stfle[0] |= S390X_CAPBIT(0);
 
-#ifdef OSSL_IMPLEMENT_GETAUXVAL
+#if defined(OSSL_IMPLEMENT_GETAUXVAL) \
+    && defined(HWCAP_S390_STFLE) \
+    && defined(HWCAP_S390_VX)
```

Comment:
i didnt think about it in my last review, but actually the checks for the defines has to be moved up to the conditional definition of OSSL_IMPLEMENT_GETAUXVAL ie: define OSS__IMPLEMENT_GETAUXVAL if and only if all compile time requirements for getauxval code are met (including the 2 defines are available).

Reply:
Fixup pushed, please look